ui/panels-plugin/src/plugins/trace-table/trace-table-model.ts

```diff
@@ -11,16 +11,29 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import { MouseEvent } from 'react';
+
 /**
  * The Options object type supported by the TraceTable panel plugin.
  */
 // Note: The interface attributes must match cue/schemas/panels/trace-table/trace-table.cue
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
```

Comment:
`// eslint-disable-next-line` can be remove if we keep it like that (depends on what is done for next comment 😄 )